Ports/zig/patches/0012-Implement-SerenityOS-support-in-std.patch

```diff
@@ -410,6 +421,7 @@ index 0000000000000000000000000000000000000000..1c8dc37af9d528338d2c5543b5f3a478
 +    pub const END = SerenityConstants.SEEK_END;
 +};
 +
++// see Kernel/API/POSIX/signal_numbers.h
```

Comment:
Ideally, the Zig stdlib should only know about constants as we expose them through libc, because libc is the "stable" ABI for the SerenityOS target. Is there a particular reason we're adding these comments here?

Reply:
I found it helpful to know where especially the structs are originating from. Thats why I put that information in the comments.

Reply:
Looks fine to me, signal stuff ends up being exposed via LibC, but the definition is Kernel-common, so that's what the comment points at (for easier referencing).